src/main/java/modify_task_use_case/ModifyTaskController.java

```diff
@@ -3,20 +3,31 @@
  * Author: Raghav Arora
  * Modified By: Raghav Arora
  * Created: Nov 12, 2022
- * Last Modified: Nov 19, 2022
+ * Last Modified: Dec 1, 2022
  */
 package modify_task_use_case;
 
 public class ModifyTaskController {
-    private ModifyTaskInputBoundary inputBoundary;
+    private final ModifyTaskInputBoundary inputBoundary;
 
+    /**
+     * @param inputBoundary instance which is used to invoke the application logic
+     * for the yse case. Instance is implicitly of type ModifyTaskInteractor.
+     */
     public ModifyTaskController(ModifyTaskInputBoundary inputBoundary) {
         this.inputBoundary = inputBoundary;
     }
 
-    ModifyTaskOutputData modifyTask(int dayId, String newTitle, String title) {
+    /**
```

Comment:
Since this is a public method it might be pertinent to include a brief description of the method. It is quite obvious what the method is doing so this should be quite short. Same thing applies to other public methods.

Reply:
Noted. Will fix that in my next commit.